Lib/mutatorMath/ufo/instance.py

```diff
@@ -197,6 +199,63 @@ def addInfo(self, instanceLocation=None, sources=None, copySourceName=None):
             copySourceObject, loc = sources[copySourceName]
             self._copyFontInfo(self.font.info, copySourceObject.info)
 
+    def _roundSelectedFontInfo(self, fontInfo):
+        """self.roundGeometry is false. However, most FontInfo fields have to be integer.
+        Exceptions are:
+            any of the PostScript Blue values.
+            postscriptStemSnapH, postscriptStemSnapV, postscriptSlantAngle
+        The Blue values should be rounded to 2 decimal places, with the exception
+        of postscriptBlueScale.
+        I round the float values because most Type1/Type2 rasterizers store
+        point and stem coordinates as Fixed numbers with 8 bits. You end up
+        cumulative rounding errors if you pass in relative values with more
+        than 2 decimal places. Other FontInfo float values are stored as Fixed
+        number with 16 bits,and can support 6 decimal places.
+        """
+        listType = type([])
+        strType = type("")
+        
+        fiKeys = dir(fontInfo)
+        for fieldName in fiKeys:
+            if fieldName.startswith("_"):
+                continue
+            srcValue = getattr(fontInfo, fieldName)
+            if type(srcValue) != listType:
+                try:
+                    realValue = float(srcValue)
+                except (TypeError, ValueError):
+                    continue
+                intValue = int(realValue)
+                if intValue == realValue:
+                    continue
+                
+                if ("Blue" in fieldName) or (fieldName == "postscriptSlantAngle"):
```

Comment:
That will work for current implementations of the font info object but can cause unexpected results of someone adds something with 'Blue' in an attribute name. Would it be better to be very specific about the names of the attributes that qualify?